src/index_notation/index_notation.cpp

```diff
@@ -2765,4 +2766,37 @@ IndexStmt zero(IndexStmt stmt, const std::set<Access>& zeroed) {
   return Zero(zeroed).rewrite(stmt);
 }
 
+IndexStmt generatePackStmt(TensorVar tensor, 
+                           std::string otherName, Format otherFormat, 
+                           std::vector<IndexVar> indexVars, 
```

Comment:
Instead of taking `indexVars` as an argument, can't the function simply instantiate the required index variables based on the order of `tensor`? Same with `generatePackCOOStmt`.

Reply:
The reason I have this parameter is so that the index variables used in the generated pack statement are consistent with the index variables used in the original input statement, which I think makes the resulting function easier to read? 

(The code for packing in the helper functions, for example, randomly generates index variables that are quite messy looking, i.e. approximately of the form i12345.)

Reply:
Ah I see, that makes sense.